_posts/2019-10-17-how-to-achieve-cis-benchmark-compliance.adoc

```diff
@@ -842,130 +857,683 @@ If you need to brush up on how the IaC Library works, read the
+https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_mfa_enable.html[enable MFA for your IAM user].
+Feel free to use a virtual or hardware MFA device—whichever is easier or required by your company—as either one
+dramatically improves the security of your IAM user. Note that using SMS (text messages) for MFA is
+https://www.schneier.com/blog/archives/2016/08/nist_is_no_long.html[no longer recommended by NIST] due to known
+https://www.theverge.com/2017/9/18/16328172/sms-two-factor-authentication-hack-password-bitcoin[vulnerabilities with the cellular system],
+so using a virtual or hardware MFA device is preferable; that said, MFA with SMS is still better than no MFA at all.
+##
 
-[[create_password_policy]]
-=== Create an IAM user password policy
-After the manual steps are complete, the next step is to create an IAM user password policy using the
-link:https://github.com/gruntwork-io/terraform-aws-cis-service-catalog/blob/master/examples/iam-password-policy/README.adoc[`iam-password-policy`
 wrapper module]. Complete this step before creating any IAM users!
+=== Configure the security baseline for the root account
+
+Next, we'll configure a security baseline for the root account that is responsible for creating all the child accounts.
+It will also configure AWS Organizations, IAM Roles, IAM Users, IAM Groups, IAM Password Policies, Amazon GuardDuty, AWS CloudTrail, AWS Config and Security Hub.
 
-[[wrapper_code]]
-==== Using the wrapper module
-This section demonstrates how to use the CIS compliance password policy wrapper module to create a password policy. Follow this pattern for each of the wrapper modules discussed throughout this walkthrough.
+We'll be using the `account-baseline-root` module from https://github.com/gruntwork-io/terraform-aws-cis-service-catalog[terraform-aws-cis-service-catalog].
 
-===== Using the wrapper module with Terraform
-If you're using Terraform without Terragrunt, use this section to create a module in your `infrastructure-modules` repository.
+[.exceptional]
+IMPORTANT: You must be a [js-subscribe-cta]#Gruntwork subscriber# to access `terraform-aws-service-catalog`.
+
+First, create a _wrapper module_ called `account-baseline-root` in your `infrastructure-modules` repo under the `landingzone` subdirectory:
 
 ----
 infrastructure-modules
-└── security
-    └── iam-password-policy
-        ├── main.tf
-        └── variables.tf
+  └ landingzone
+    └ account-baseline-root
+      └ main.tf
+      └ outputs.tf
+      └ variables.tf
 ----
 
-Inside of `main.tf`, configure your AWS provider settings:
+Inside of `main.tf`, configure your AWS provider and Terraform settings:
 
-.infrastructure-modules/security/iam-password-policy/main.tf
+.infrastructure-modules/landingzone/account-baseline-root/main.tf
 [source,hcl]
 ----
 provider "aws" {
+  # The AWS region in which all resources will be created
   region = var.aws_region
+
+  # Require a 3.x version of the AWS provider
+  version = "~> 3.40"
```

Comment:
I've upped the version of the AWS provider here because that's what we seem to be doing the RefArches with - e.g. Nomisma `lock.hcl` file

Reply:
Makes sense.